misc/python/materialize/test_analytics/setup/views/30-ci-failures.sql

```diff
@@ -0,0 +1,45 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Use of this software is governed by the Business Source License
+-- included in the LICENSE file at the root of this repository.
+--
+-- As of the Change Date specified in that file, in accordance with
+-- the Business Source License, use of this software will be governed
+-- by the Apache License, Version 2.0.
+
+CREATE OR REPLACE MATERIALIZED VIEW mv_ci_failures AS
+    SELECT
+        pipeline || '#' || build_number AS build,
+        test_suite,
+        CASE WHEN error_type = 'KNOWN_ISSUE' THEN issue ELSE error_type END AS issue,
+        string_agg(content, '\n') AS content,
+        branch,
+        build_date,
+        mz_version,
+        commit_hash,
+        build_step_key,
+        test_retry_count,
+        occurrence_count,
+        build_id,
+        build_job_id
+    FROM v_build_annotation_error
+    GROUP BY
```

Comment:
I don't get the aggregation here? Do we group by all fields except `content`? Why? And don't we need to use `sum(occurrence_count)`?

Reply:
Yes, I just wanted to catch the (relatively common) case where there are multiple errors in the same run.